manim/mobject/text/tex_mobject.py

```diff
@@ -184,13 +188,13 @@ def _modify_special_strings(self, tex):
                 tex = ""
         return tex
 
-    def _remove_stray_braces(self, tex):
-        r"""
-        Makes :class:`~.MathTex` resilient to unmatched braces.
+    def _remove_stray_braces(self, tex_string: str) -> str:
+        r"""Makes :class:`~.MathTex` resilient to unmatched braces.
 
         This is important when the braces in the TeX code are spread over
         multiple arguments as in, e.g., ``MathTex(r"e^{i", r"\tau} = 1")``.
         """
+        tex = tex_string
```

Comment:
Wouldn't it be better just to leave the method signature as `(self, tex: str)`?

Reply:
I wanted it to be consistent with the other methods where that parameter was also names `tex_string`.